### PR TITLE
Replace job log box with container table

### PR DESCRIPTION
**Pull request by willie-yao**
This PR replaces the log box on the job detail page with a table listing all of the job containers

### CODE REVIEW

**Comment by willie-yao** [comment]
@krancour Any suggestions on what to display for each container? The containers don't seem to have a name field, but has an Image, ImagePullPolicy, Command, Arguments, and Environment fields

**Comment by krancour** [comment]
At least image.

We'll figure out what else later.

I think it's inevitable that we'll _eventually_ add a feature that lets you see the raw json (or more preferably yaml) for a given thing-- be it project, event, job, whatever.

File: v2/cli/term/job_page.go [hunk 1]
```diff
@@ -103,62 +98,87 @@ func (j *jobPage) fillJobInfo(job core.Job) {
...
-	// j.canStream = make(chan struct{})
-
-	// // Save the context on goroutine.
-	// ss := j.stopStreaming
-	// cs := j.canStream
-	// l := ctx.Log
-
-	// // Close our reader when finished streaming, ignore if error.
-	// defer l.Close()
+func (j *jobPage) fillContainerTable(eventID string, job core.Job) {
```
Comment (krancour) [inline v2/cli/term/job_page.go]: Nit: The UI control in question is named `containersTable` (plural "containers"), so I'd slightly prefer that the function is named `fillContainersTable` for consistency.

Comment (krancour) [reply v2/cli/term/job_page.go]: fwiw... remember "nits" mean the reviewer is acknowledging just how picky they are being. Nits don't generally block your PR from being merged.

**Pull request merged**